ospf6d/ospf6_area.c

```diff
@@ -703,12 +705,32 @@ DEFUN (no_area_filter_list,
 				return CMD_SUCCESS;
 
 		XFREE(MTYPE_OSPF6_PLISTNAME, PREFIX_NAME_OUT(area));
-		ospf6_abr_enable_area(area);
+		PREFIX_LIST_OUT(area) = NULL;
```

Comment:
Is this required here ?

Reply:
I'm just clearing the area filter-out pointer if the filter is removed.  In ospf6_abr.c, ospf6_abr_originate_summary_to_area it does a check if this field is null or if not it tries to apply the filter.

	/* Check filter-list */
	if (PREFIX_LIST_OUT(area))
		if (prefix_list_apply(PREFIX_LIST_OUT(area), &route->prefix)
		    != PREFIX_PERMIT) {

Reply:
Okay, got you. 
Thanks